content/docs/other-guides/kubeflow-on-multinode-cluster.md

```diff
@@ -141,85 +141,46 @@ To perform this task you need to:
 2. Change their storage class.
 3. Delete and recreate them in the cluster.
 
-Download the four PVCs:
+For downloading the existing PVCs, first get list of all PVCs:
 
 ```shell
-kubectl get pvc/mysql-pv-claim -n kubeflow -o yaml > mysql-pv-claim.yaml
-kubectl get pvc/minio-pv-claim -n kubeflow -o yaml > minio-pvc.yaml
-kubectl get pvc/katib-mysql -n kubeflow -o yaml > katib.yaml
-kubectl get pvc/metadata-mysql -n kubeflow -o yaml > metadata.yaml
+kubectl get pvc --all-namespaces
 ```
 
-And then modify files to add the right `storageClassName` under the `spec` section:
+This will show the list of PVCs with details including `NAMESPACE`, `STATUS`, `STORAGECLASS`.
+(e.g. `mysql-pv-claim` in `Kubeflow` namespace and `authservice-pvc` in `istio-systems` namespace).
 
-```yaml
-# mysql-pv-claim.yaml
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: mysql-pv-claim
-  namespace: kubeflow
-  ...
-spec:
-  storageClassName: nfs
-  ...
-```
+Download the PVCs:
 
-```yaml
-# minio-pvc.yaml
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: minio-pv-claim
-  namespace: kubeflow
-  ...
-spec:
-  storageClassName: nfs
-  ...
+```shell
+kubectl get pvc/<PVC-NAME> -n <NAMESPACE> -o yaml > <PVC-NAME>.yaml
 ```
 
-```yaml
-# katib.yaml
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: katib-mysql
-  namespace: kubeflow
-  ...
-spec:
-  storageClassName: nfs
-  ...
-```
+And then modify files to add the right `storageClassName` under the `spec` section:
```

Comment:
For clarity, change this:

```
And then modify files
```

To this:
```
And then modify the YAML files
```
Also, add "For example":

```
under the `spec` section. For example:
```

Reply:
done. thank you @sarahmaddox